adala/skills/collection/entity_extraction.py

```diff
@@ -32,13 +32,14 @@ def validate_output_format_for_ner_tag(
             continue
         text = row[input_field_name]
         entities = row[output_field_name]
-        for entity in entities:
-            corrected_text = text[entity["start"] : entity["end"]]
-            if entity.get("text") is None:
-                entity["text"] = corrected_text
-            elif entity["text"] != corrected_text:
-                # this seems to happen rarely if at all in testing, but could lead to invalid predictions
-                logger.warning(f"text and indices disagree for a predicted entity")
+        if entities is not None:
+            for entity in entities:
+                corrected_text = text[entity["start"] : entity["end"]]
+                if entity.get("text") is None:
+                    entity["text"] = corrected_text
+                elif entity["text"] != corrected_text:
+                    # this seems to happen rarely if at all in testing, but could lead to invalid predictions
+                    logger.warning(f"text and indices disagree for a predicted entity")
```

Comment:
should we log this as an error, so we have it surfaced in sentry?

Reply:
Sure, good idea

Reply:
Changed. Not logging the actual text to avoid leaking customer data, but not sure how big of a concern that is, can change later if we need to for debugging